labs/lab-10/tasks/overflow-for-binary/tests/Makefile

```
@@ -0,0 +1,24 @@
+CC     = gcc
+CFLAGS = -Wall -Wextra -O2
+TARGET = test_overflow_for_binary
+
+SRCS = test_overflow_for_binary.c graded_test.c
+
+OBJS = $(SRCS:.c=.o)
+
+all: $(TARGET)
+
+$(TARGET): $(OBJS)
+	$(CC) $(CFLAGS) $(OBJS) -o $@
+
+%.o: %.c
+	$(CC) $(CFLAGS) -c $< -o $@
+
+clean:
+	rm -f $(TARGET) $(OBJS)
+
+check: all
+	chmod +x run_all_tests.h
+	./run_all_tests.h
+
+.PHONY: all clean test
```

```suggestion
.PHONY: all clean test

```
Make sure all makefiles end in a newline.